package/tests/test_api/test_graphql.py

```diff
@@ -87,6 +87,52 @@ def example_multiple_run_tracking_catalog(save_version, save_new_version):
 
     shutil.rmtree("test.json", ignore_errors=True)
 
+@pytest.fixture
+def example_multiple_run_tracking_catalog_one_empty_run(save_version, save_new_version):
+    # Note - filepath is assigned without using tmp_path as it fails on windows build.
+    # This is a temp soln and will be cleaned up in the future.
+    new_metrics_dataset = MetricsDataSet(
+        filepath="test.json",
+        version=Version(None, save_version),
+    )
+    new_metrics_dataset.save({"col1": 1, "col3": 3})
+    new_metrics_dataset = MetricsDataSet(
+        filepath="test.json",
+        version=Version(None, save_new_version),
+    )
+    catalog = DataCatalog(
+        data_sets={
+            "new_metrics": new_metrics_dataset,
+        }
+    )
+
+    yield catalog
+
+    shutil.rmtree("test.json", ignore_errors=True)
+
+
+@pytest.fixture
+def example_multiple_run_tracking_catalog_both_runs_empty(save_version, save_new_version):
+    # Note - filepath is assigned without using tmp_path as it fails on windows build.
+    # This is a temp soln and will be cleaned up in the future.
+    new_metrics_dataset = MetricsDataSet(
+        filepath="test.json",
+        version=Version(None, save_version),
+    )
+    new_metrics_dataset = MetricsDataSet(
+        filepath="test.json",
+        version=Version(None, save_new_version),
+    )
```

Comment:
Why are there 2 objects with the same name here?

Reply:
I am trying to create two runs for the same metrics -- using two versions save_version, and save_new_version.